packages/office-addin-lint/src/cli.ts

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import * as commander from "commander";
+import { logErrorMessage } from "office-addin-cli";
+import * as commands from "./commands";
+import * as defaults from "./defaults";
+
+commander.name("office-addin-lint");
+commander.version(process.env.npm_package_version || "(version not available)");
+
+commander
+    .command("lint")
+    .option("--files <files>", `Path to files to perform lint check. Default: ${defaults.lintFiles}`)
```

Comment:
`Path to files to perform lint check.` => `Specifies the source files to check.`  (keep the default)